gluster-exporter/metric_brick.go

```diff
@@ -131,14 +132,14 @@ func diskUsage(path string) (disk DiskStatus, err error) {
 func brickUtilization() {
 	volumes, err := glusterutils.VolumeInfo(&glusterConfig)
 	if err != nil {
-		// TODO: Log error
+		log.WithError(err).Fatal("Error getting volume info")
```

Comment:
`.Fatal` will exit the process. We have to continue even if there are failures(Temporary failures will be resolved in the next iteration). Also crashing the exporter on one metrics collector's failure is not good thing.

Reply:
In that case should I use Warn in these cases? I feel yes.